client-src/elements/chromedash-drawer.ts

```diff
@@ -295,6 +295,7 @@ export class ChromedashDrawer extends LitElement {
         ${this.renderNavItem('/metrics/css/popularity', 'CSS')}
         ${this.renderNavItem('/metrics/css/animated', 'CSS Animation')}
         ${this.renderNavItem('/metrics/feature/popularity', 'JS/HTML')}
+        ${this.renderNavItem('/metrics/webfeature/popularity', 'Web features')}
```

Comment:
```suggestion
        ${this.renderNavItem('/metrics/webfeature/popularity', 'Web Features')}
```
`CSS Animation` are also all caps. Ditto here?